File: VNNLanguage/Model/Character.cs
using System;
using System.Drawing;

namespace VNNLanguage
{
    public class Character
    {
        public string FriendlyName { get; set; }

        public string DisplayName { get; set; }

        public Coordinate Position { get; set; }

        public int SpriteHeight { get; set; }

        public int SpriteWidth { get; set; }

        public Bitmap CurrentSprite { get; set; }

        public bool InScene { get; set; }

        public Guid Identifier { get; set; }
    }

}

File: SM-ElectronContainer/src/models/Character.ts
import { Coordinate } from './Coordinate';

export class Character {
  public Identifier: string = '';
  public FriendlyName: string = '';
  public DisplayName: string = '';
  public Position: Coordinate = new Coordinate();
  public SpriteHeight: number = 50;
  public SpriteWidth: number = 50;
  public CurrentSprite: string = '';
  public InScene: boolean = false;
}
